src/components/EditItemsList/EditItemsList.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useRef } from 'react';
+import {
+  Wrap,
+  Input,
+  Box,
+  Button,
+  Flex,
+  FormControl,
+  FormLabel,
+  FormErrorMessage,
+  FormHelperText,
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'FormHelperText' is defined but never used.